src/.env.sample

```diff
@@ -221,3 +221,6 @@ PROGRAM_PUBLISH_KAFKA_TOPIC=dev.programpublish
 
 # Marks the highest level in the entity-hierarchy
 HIGHEST_IN_ENTITY_HIERARCHY="state"
+
+#Add the deep link url here
+PROJECT_DEEP_LINK_URL="https://host.domain/view/project/"
```

Comment:
@adithyadinesh0412  this can be localhost or project port if view/project is an api end point